jsapp/js/components/modalForms/submission.es6

```diff
@@ -105,12 +106,16 @@ class Submission extends React.Component {
         hasBetaQuestion: hasBetaQuestion
       });
     }).fail((error)=>{
-      if (error.responseText)
-        this.setState({error: error.responseText, loading: false});
-      else if (error.statusText)
-        this.setState({error: error.statusText, loading: false});
-      else
+      if (error.responseText) {
+        let error_message = error.responseText;
+        if (error_message === DETAIL_NOT_FOUND)
+          error_message = 'The submission could not be found. It may have been deleted. Submission ID: ' + sid;
```

Comment:
Plese make it a translatable string: `error_message = t('The submission could not be found. It may have been deleted. Submission ID: ##id##').replace('##id##', sid);`